lansa-vmss-windows-autoscale-sql-database/createUiDefinition.json

```diff
@@ -622,36 +660,66 @@
                             "required": true,
                             "regex": "^(?!\\s*$).+",
                             "validationMessage": "The value must not be empty."
-                        }
+                        },
+                        "visible": "[not(and(or(equals(steps('applicationGatewayConfig').applicationGatewaySkuTier, 'Standard_v2'), equals(steps('applicationGatewayConfig').applicationGatewaySkuTier, 'WAF_v2')), equals(steps('applicationGatewayConfig').autoScalingEnabled, true)))]"
                     },
                     {
-                        "name":"publicIpAddress",
-                        "type": "Microsoft.Network.PublicIpAddressCombo",
-                        "label": {
-                            "publicIpAddress": "Application Gateway Public IP address",
-                            "domainNameLabel": "Domain name label"
-                        },
-                        "toolTip": {
-                           "publicIpAddress": "Whether to use an existing public IP Address or create a new one",
-                           "domainNameLabel": "Domain name label"
-                        },
-                        "defaultValue": { 
-                            "publicIpAddressName": "[toLower(concat(basics('stackName'),'-agpip'))]",
-                            "domainNameLabel": ""
+                        "name": "autoScaleMinCapacity",
+                        "type": "Microsoft.Common.TextBox",
+                        "label": "Minimum scale units",
+                        "toolTip": "Additional reserved capacity which is always maintained irrespective of traffic. 0 signifies purely autoscaling Application Gateway with no additional reserved capacity while still maintaining high availability of the service.",
+                        "defaultValue": "0",
+                        "constraints": {
+                            "required": true,
+                            "regex": "^(?!\\s*$).+",
+                            "validationMessage": "The value must not be empty."
                         },
+                        "visible": "[equals(steps('applicationGatewayConfig').autoScalingEnabled, true)]"
+                    },
+                    {
+                        "name": "autoScaleMaxCapacity",
+                        "type": "Microsoft.Common.TextBox",
+                        "label": "Maximum scale units",
+                        "toolTip": "Upper bound on number of Application Gateway capacity",
```

Comment:
"Upper bound on Application Gateway capacity"

Reply:
Will add it.